### PR TITLE
Governance Documentation For Sandbox Requirements

### PR DESCRIPTION
**Pull request by smcclure20**
Initial governance documentation for discussion by the TSC.

Mostly copies boilerplate formatting from the L3af project

### CODE REVIEW

File: governance/roles.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+TSC membership is documented [here](tsc.md).
+
+## MAC Liason
+TODO
+
+## TAC Liason
+TODO
+
+## Release Manager
+Primary: TODO
```
Comment (praveingk) [inline governance/roles.md:13]:
```suggestion
Primary: Pravein Govindan Kannan
```

File: governance/release-process.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+[GitHub Issues](https://github.com/paraglider-project/paraglider/issues).
+
+## Steps to Create a New Release (TODO: Review this more closely)
+1. Bump the version number ([Semantic Versioning 2.0](https://semver.org/))
+  for the following files:
+  * [version.go](../internal/version/version.go)
+2. Create a new pull request for the version number changes
+3. Once the build artifacts complete and all checks pass, download the build
+  artifacts, which can be found via the Actions tab on github
+4. Copy the binary into a VM and manually test user functionality (use the CLI, client, etc.). TODO: Add to CI/CD
```
Comment (praveingk) [inline governance/release-process.md:29]: As part of step 2, we should have the CICD in place. at least for VMs. Maybe we strengthen that, then step 4 would be redundant.

File: governance/release-process.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  (e.g., "git tag v1.0.0", "git push --tags")
+6. Navigate to the repository and find where it says
+  "Releases". Click the "Create a new release" hyperlink
+7. Click the "Choose a tag" dropdown and select the tag created in the previous
+  step
+8. Enter a "Release title" and click on "Generate release notes" button to pull
+  in all the PR changes since the last tagged release
+9. Attach binaries / artifacts to the release by clicking on the "Attach
+  binaries..." text. These are the same artifacts downloaded above
+10. After binaries are uploaded, click the "Publish release" button to post the
```
Comment (praveingk) [inline governance/release-process.md:40]: Maybe, we do a dry run for v0.0.2 release to capture the steps more accurately.

File: governance/release-process.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+## Release Tracking
+
+Feature work for upcoming releases is tracked through
+[GitHub Issues](https://github.com/paraglider-project/paraglider/issues).
+
+## Steps to Create a New Release (TODO: Review this more closely)
+1. Bump the version number ([Semantic Versioning 2.0](https://semver.org/))
+  for the following files:
+  * [version.go](../internal/version/version.go)
```
Comment (praveingk) [inline governance/release-process.md:25]: Version must be updated in ../build/version.mk

File: governance/release-process.md (hunk 3 above)
Comment (praveingk) [reply governance/release-process.md:40]: I just tried this, without actually publishing the release.  The steps 5-10 are ok 👍🏽

File: governance/release-process.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+9. Attach binaries / artifacts to the release by clicking on the "Attach
+  binaries..." text. These are the same artifacts downloaded above
+10. After binaries are uploaded, click the "Publish release" button to post the
+  release
+
+## Stable Releases
+
+Stable releases of Paraglider include:
+
+* Maintenance window (any version released in the last 6 to 12 months).
```
Comment (divega) [inline governance/release-process.md:47]: It seems to me that we will only have stable releases once we have reached v1. If yes, we possibly need to say that explicitly somewhere to differentiate from what is right now in the repo is not considered stable. 

Also, should we just say 12 months here and delete 6?

File: governance/release-process.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+* Stability fixes backported from the `main` branch (anything that can result in
```
Comment (divega) [inline governance/release-process.md:48]:
```suggestion
* Stability fixes backported from the `main` branch (fixing anything that can result in
```

Comment (divega) [inline governance/release-process.md:48]: Nit: For authoring markdown, I have found that keeping sentences together in their own separate lines makes for easier long-term changeability/diffing.

E.g., in the case, I wouldn't have a line brake after "result in". In paragraph with multiple sentences, I would add a line break between sentences. Markdown rendering automatically stitches sentences back together to form a paragraph and only renders paragraph breaks when there is a double line break (or two spaces) after a sentence.

File: governance/roles.md (hunk 1 above)
Comment (divega) [reply governance/roles.md:13]: You have my vote :smile:

File: governance/tsc-procedures.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
+# Technical Steering Committee (TSC) Procedures
+## TSC Membership
+Membership changes to the Paraglider TSC is governed by the current list of TSC members with voting rights. TSC membership changes require a majority vote by the current TSC members in order to pass. For more details on TSC voting procedures please refer to the Paraglider Technical Charter "TSC Voting" section.
+
+## Adding TSC Members
+In order to have a vote on adding a new member to the Paraglider TSC a github issue should be opened against the repository and tagged with the “governance” label. The title of the issue should be TSC New Member - <NAME> with a description on why this person should be added to the TSC. The opened github issue should then be referenced in the next TSC meeting which will determine a date to conduct a vote on the new membership. If a vote does not occur within a 1-month period from opening the issue the motion will be considered denied, and the issue will be closed.
```
Comment (divega) [inline governance/tsc-procedures.md:6]: Assuming you want \<NAME> to be visible, I think you need to escape at least the opening angle bracket in markdown.

```suggestion
In order to have a vote on adding a new member to the Paraglider TSC a github issue should be opened against the repository and tagged with the “governance” label. The title of the issue should be TSC New Member - \<NAME> with a description on why this person should be added to the TSC. The opened github issue should then be referenced in the next TSC meeting which will determine a date to conduct a vote on the new membership. If a vote does not occur within a 1-month period from opening the issue the motion will be considered denied, and the issue will be closed.
```

Comment (divega) [reply governance/tsc-procedures.md:6]: Also, an example of how I would format long paragraphs to optimize for changeability and diffing:

```suggestion
In order to have a vote on adding a new member to the Paraglider TSC a github issue should be opened against the repository and tagged with the “governance” label.
The title of the issue should be TSC New Member - \<NAME> with a description on why this person should be added to the TSC.
The opened GitHub issue should then be referenced in the next TSC meeting which will determine a date to conduct a vote on the new membership.
If a vote does not occur within a 1-month period from opening the issue the motion will be considered denied, and the issue will be closed.
```

File: governance/tsc-procedures.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+
+## Removal of TSC Members
+To request removal of a TSC member open a github issue with the “governance” label and with the title TSC "Membership Removal - <NAME>".
```
Comment (divega) [inline governance/tsc-procedures.md:9]:
```suggestion
To request removal of a TSC member open a GitHub issue with the “governance” label and with the title TSC "Membership Removal - \<NAME>".
```

File: governance/tsc-procedures.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+
+Existing TSC members can request their own removal from the Paraglider TSC at anytime and the issue will be taken up in the next weekly TSC meeting. The person will then be removed as a TSC member. If there is a proposed replacement for the member leaving, they should be mentioned in the removal issue. The proposed replacement member should open a new github issue and follow the Adding TSC Members section.
+
+Removal of a TSC member can be requested by other TSC members or from the community with a description on why they should be removed listed in the issue. Examples of reasons for removal could be lack of activity of that TSC member for a prolonged period of time. Requests for removal will be discussed in the next weekly TSC meeting and a date will be determined for a vote on the membership removal.
+
+Please DO NOT report Code of Conduct issues here and instead follow the reporting procedures defined in the Paraglider Project Code of Conduct.
```
Comment (divega) [inline governance/tsc-procedures.md:15]:
```suggestion
Please DO NOT report Code of Conduct issues in the issue and instead follow the reporting procedures defined in the Paraglider Project Code of Conduct.
```

**Review by divega — APPROVED** [review]
LGTM in general, just a few minor suggestions.

**Review by seankimkdy — APPROVED** [review]

**Pull request merged**